application/templates/datamanager/assign-entities-check-results.html

```
@@ -0,0 +1,388 @@
+          {% else %}
+          Not available
+          {% endif %}
+        </dd>
+      </div>
+      <div class="govuk-summary-list__row">
+        <dt class="govuk-summary-list__key">Endpoints</dt>
+        <dd class="govuk-summary-list__value">
+          {% if existing_endpoints %}
+            {% for endpoint in existing_endpoints %}
+              {% if endpoint["endpoint-url"] %}
+              <div><a href="{{ endpoint["endpoint-url"] }}" target="_blank" style="word-break: break-all;">{{ endpoint["endpoint-url"] }}</a></div>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ < ].

(spec-char-escape)